tests/fixtures/RTCIceCandidate_candidate.json

```diff
@@ -1,3 +1,6 @@
 {
-  "candidate": "3932168448 1 udp 1694498815 1.2.3.4 10676 typ srflx raddr 0.0.0.0 rport 37566"
+  "candidate": "3932168448 1 udp 1694498815 1.2.3.4 10676 typ srflx raddr 0.0.0.0 rport 37566",
+  "sdpMLineIndex": 0,
```

Comment:
Please revert changes to this file and it must be valid to keep backwards compability

Reply:
Done